src/LibVLCSharp.Forms/Shared/Themes/Generic.xaml

```diff
@@ -337,6 +337,11 @@
                     <ContentView HeightRequest="300" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" x:Name="TracksOverlayView" AbsoluteLayout.LayoutBounds="0, 0, 1, 1"  AbsoluteLayout.LayoutFlags="All" IsVisible="False" BackgroundColor="#7F000000">
                         <ScrollView VerticalScrollBarVisibility="Never" HorizontalOptions="FillAndExpand" VerticalOptions="Fill" Orientation="Vertical">
                             <Grid>
+                                <Grid.RowDefinitions>
```

Comment:
Isn't there a StackPanel on X.F? wouldn't that work here?

Reply:
With the StackLayout, we will need to define its height, which can cause issues because we do not know in advance the number of tracks in every ListView.
So the Grid is more suitable. The listViews will occupy heights according to the number of tracks in their ItemSource.